compiler/env/OMRKnownObjectTable.cpp

```diff
@@ -67,6 +67,25 @@ OMR::KnownObjectTable::getIndex(uintptr_t objectPointer, bool isArrayWithConstan
    return index;
    }
 
+TR::KnownObjectTable::Index
+OMR::KnownObjectTable::getOrCreateIndex(uintptr_t objectPointer)
+   {
+   TR_UNIMPLEMENTED();
+   return -1;
+   }
+
+TR::KnownObjectTable::Index
+OMR::KnownObjectTable::getOrCreateIndex(uintptr_t objectPointer, bool isArrayWithConstantElements)
+   {
+   TR_ASSERT(TR::Compiler->vm.hasAccess(self()->comp()), "Getting KnownObjectTable index requires VM access");
+   TR::KnownObjectTable::Index index = self()->getOrCreateIndex(objectPointer);
+   if (isArrayWithConstantElements)
+      {
+      self()->addArrayWithConstantElements(index);
+      }
+   return index;
+   }
```

Comment:
Same comment as for the others.

Reply:
same answer as `getOrCreateIndexAt()` the above.